test/e2e/node/kubelet_authz.go

```diff
@@ -88,17 +87,27 @@ func runKubeletAuthzTest(ctx context.Context, f *framework.Framework, endpoint,
 		},
 	}, metav1.CreateOptions{})
 	framework.ExpectNoError(err)
+	defer func() {
+		ginkgo.By(fmt.Sprintf("Destroying ClusterRoles %q for this suite.", clusterRole.Name))
+		framework.ExpectNoError(f.ClientSet.RbacV1().ClusterRoles().Delete(ctx, clusterRole.Name, metav1.DeleteOptions{}))
+	}()
 
 	subject := rbacv1.Subject{
 		Kind:      rbacv1.ServiceAccountKind,
 		Namespace: ns,
 		Name:      saName,
 	}
 
-	ginkgo.By(fmt.Sprintf("Creating ClusterRoleBinding with ClusterRole %s with subject %s/%s", crName, ns, saName))
+	ginkgo.By(fmt.Sprintf("Creating ClusterRoleBinding with ClusterRole %s with subject %s/%s", clusterRole.Name, ns, saName))
 
-	err = e2eauth.BindClusterRole(ctx, f.ClientSet.RbacV1(), crName, ns, subject)
+	clusterRoleBinding, err := e2eauth.BindClusterRole(ctx, f.ClientSet.RbacV1(), clusterRole.Name, ns, subject)
 	framework.ExpectNoError(err)
+	defer func() {
+		if clusterRoleBinding != nil {
```

Comment:
what case is this possible? would we ever not error and also have a nil clusterRoleBinding here?

Reply:
@haircommander when RBAC is not enabled, [here](https://github.com/kubernetes/kubernetes/pull/131249/files#diff-ccf4cfda4ae4a3d8ea544602dd59220ff5a8fdb2e09e02a9d6beda359ab063d9R107-R108).

Reply:
:facepalm: clear as day!